pkg/KubeArmorAnnotation/config/manager/manager.yaml

```diff
@@ -22,6 +22,7 @@ spec:
       annotations:
         container.apparmor.security.beta.kubernetes.io/manager: unconfined
       labels:
+        kubearmor-app: annotation-manager
```

Comment:
rather than updating this file, can you update [kubearmor-annotation-manager.yaml](https://github.com/kubearmor/KubeArmor/blob/main/deployments/annotations/kubearmor-annotation-manager.yaml)
"control-plane: controller-manager" -> "kubearmor-app: annotation-manager"

In addition, while updating the YAML file, can you remove the namespace part?
```
apiVersion: v1
kind: Namespace
metadata:
  labels:
    control-plane: controller-manager
  name: kube-system
```